service/controller/chart/v1/resource/release/create_test.go

```diff
@@ -75,6 +75,8 @@ func Test_Resource_Release_newCreate(t *testing.T) {
 			HelmClient: helmclienttest.New(helmclienttest.Config{}),
 			K8sClient:  k8sfake.NewSimpleClientset(),
 			Logger:     microloggertest.New(),
+
+			ProjectName: "chart-operator",
```

Comment:
It would be good to introduce `pkg/project` here https://github.com/giantswarm/aws-operator/tree/master/pkg/project. It is automatically populated by architect.

Reply:
Good point. I did that in some other repos like pv-cleanup-operator but missed it here. I'll get it sorted.